connector/src/main/java/org/jboss/as/connector/subsystems/datasources/Constants.java

```diff
@@ -457,7 +457,7 @@ public void validateResolvedParameter(String parameterName, ModelNode value) thr
             .setXmlName(Security.Tag.AUTHENTICATION_CONTEXT.getLocalName())
             .setAllowExpression(true)
             .setRequires(ELYTRON_ENABLED_NAME)
-            .addAlternatives(SECURITY_DOMAIN_NAME, USERNAME_NAME)
+            .addAlternatives(SECURITY_DOMAIN_NAME)
```

Comment:
is this change necessary? it still shouldn't be allowed to set authentication-context and username together.